python_modules/libraries/dagster-dbt/dagster_dbt/cloud_v2/client.py

```diff
@@ -109,7 +110,11 @@ def _make_request(
                 )
                 response.raise_for_status()
                 resp_dict = response.json()
-                return resp_dict["data"] if "data" in resp_dict else resp_dict
+                return (
```

Comment:
Nitty but I don't love this; feels like there should just be a method to return the underlying response and then let the caller do whatever with it

Reply:
Yeah that's fair, I can fix this and other client methods in a separate PR